cmd/kube-scheduler/app/options/deprecated.go

```diff
@@ -87,16 +87,22 @@ func (o *DeprecatedOptions) Validate() []error {
 	return errs
 }
 
-// ApplyTo sets cfg.AlgorithmSource from flags passed on the command line in the following precedence order:
+// PolicyConfig returns whether a legacy policy config has been set
```

Comment:
Document the usage of the policy-related command line flags.

Reply:
we actually don't need this function when I made the changes from https://github.com/kubernetes/kubernetes/pull/92531#discussion_r447075338 and https://github.com/kubernetes/kubernetes/pull/92531#discussion_r447076538